skyportal/app_server.py

```diff
@@ -189,6 +190,7 @@ def log_request(self, handler):
     (r'/api/internal/notifications(/[0-9]+)?', NotificationHandler),
     (r'/api/internal/notifications/all', BulkNotificationHandler),
     (r'/api/internal/ps1_thumbnail', PS1ThumbnailHandler),
+    (r'/api/internal/recent_gcn_events', RecentGcnEventsHandler),
```

Comment:
Can you help me understand why something is internal or is not?

Reply:
Internal endpoints are those we typically only call from the front-end and don't necessarily intend to be public-facing.

Reply:
@acrellin Ok thanks!